yarn-project/circuits.js/src/structs/kernel/__snapshots__/index.test.ts.snap

```diff
@@ -113,7 +113,7 @@ value: 0xd05
 "
 `;
 
-exports[`structs/kernel serializes and prints EcdsaSignature 1`] = `"{ 0101010101010101010101010101010101010101010101010101010101010101, 0101010101010101010101010101010101010101010101010101010101010101, 0 }"`;
```

Comment:
As discussed with @PhilWindle, this final value in the snapshot here should not exist because the signature tested is not an ECDSA signature as the naming suggests--it was switched in some places to a Schnorr signature. I complete this switch by changing the definition of `abis__test_roundtrip_serialize_signature` to use a Schnorr signature and updated the snapshot accordingly.

Reply:
BTW, I tried to rename the test and ran into a weird error:
```
 FAIL  structs/kernel/index.test.ts
  ● Test suite failed to run

    ReferenceError: exports is not defined

      1 | import { expectSerializeToMatchSnapshot } from '../../tests/expectSerialize.js';
    > 2 | import { makeSchnorrSignature } from '../../tests/factories.js';
        |                       ^
      3 | import {
      4 |   makePreviousKernelData,
      5 |   makePrivateKernelInputsInner,

      at structs/kernel/index.test.ts:2:23
```